AssetInformationApi/V1/Domain/AssetManagement.cs

```diff
@@ -11,5 +11,7 @@ public class AssetManagement
         public Guid ManagingOrganisationId { get; set; }
         public string Owner { get; set; }
         public bool IsTMOManaged { get; set; }
+        public string OccupationStatus { get; set; }
+        public bool IsNoRepairsMaintenance { get; set; }
```

Comment:
If it has been created yet, you should also update the asset shared package to include these fields on the equivalent class there.

Reply:
Done :)